scripts/three-sixty.js

```diff
@@ -25,8 +25,8 @@ H5P.ThreeSixty = (function (EventDispatcher, THREE) {
    * @param {number} options.segments
    * @param {Function} [sourceNeedsUpdate] Determines if the source texture needs to be rerendered.
    */
-  function ThreeSixty(sourceElement, options, sourceNeedsUpdate) {
-    /** @alias H5P.ThreeSixty# */
+  function NDLAThreeSixty(sourceElement, options, sourceNeedsUpdate) {
+    /** @alias H5P.NDLAThreeSixty# */
```

Comment:
`NDLAThreeSixty` is contained within `H5P.NDLAThreeSixty`'s function scope here, so it's not strictly necessary to change it. Either change it back to `ThreeSixty`, or remember to change the references further down the file.

```suggestion
  function ThreeSixty(sourceElement, options, sourceNeedsUpdate) {
    /** @alias H5P.ThreeSixty# */
```